index.js

```diff
@@ -18,7 +31,7 @@ function getRuleURI(ruleId) {
 
   var pluginName = ruleParts[0];
   var ruleName = ruleParts[1];
-  var url = plugins[pluginName];
+  let url = plugins[pluginName];
```

Comment:
`let` support is [iffy at best](https://kangax.github.io/compat-table/es6/#test-let) in Node.js v4, keep this as a `var` for now.